src/test/java/nl/tudelft/ewi/devhub/webtests/views/DeliveryReviewView.java

```diff
@@ -0,0 +1,196 @@
+package nl.tudelft.ewi.devhub.webtests.views;
+
+import lombok.Data;
+import nl.tudelft.ewi.devhub.server.database.entities.Delivery;
+import nl.tudelft.ewi.devhub.server.database.entities.Delivery.State;
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+import java.util.Optional;
+
+import javax.annotation.Nullable;
+
+/**
+ * Created by Douwe Koopmans on 9-5-16.
+ */
+public class DeliveryReviewView extends ProjectSidebarView {
+
+
+    public DeliveryReviewView(WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
+    public void invariant() {
+        super.invariant();
+        Assert.assertTrue(currentPathStartsWith("/courses/ti1705/TI1705/groups/"));
```

Comment:
Will be better if we make the tests less course specific, so perhaps a `currentPathContains('assignment')